packages/puppeteer-core/src/cdp/ElementHandle.ts

```diff
@@ -162,66 +157,6 @@ export class CdpElementHandle<
     }
   }
 
-  @throwIfDisposed()
-  @ElementHandle.bindIsolatedHandle
```

Comment:
let's not introduce breaking (or any changes) to the CDP implementation? I don't want us to spend time tracing diffs in DevTools test cases

Reply:
There shouldn't be any difference in CDP. The new implementation is the same as the old one.